packages/frontend/webapp/src/js/app/pages/TownDetails/TownDetailsClosedData.vue

```diff
@@ -0,0 +1,131 @@
+                </div>
+            </DetailsPanelSection>
+            <div class="border-t-1 border-G200 py-4">
+                <table
+                    class="table-fixed text-center"
+                    v-if="town.closingSolutions.length > 0"
+                >
+                    <thead>
+                        <tr>
+                            <td class="border-b-1 font-bold">
+                                Orientations des ménages
+                            </td>
+                            <td class="w-32 py-2 border-b-1 font-bold">
+                                Ménages
+                            </td>
+                            <td class="w-32 py-2 border-b-1 font-bold">
+                                Personnes
+                            </td>
+                            <td class="w-48 py-2 border-b-1 font-bold">
+                                Message
+                            </td>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        <tr
+                            v-for="item in town.closingSolutions"
+                            :key="item.id"
+                        >
+                            <td class="text-left pr-4 border-b-1 border-r-1">
+                                {{ closingSolutions[item.id] }}
+                            </td>
+                            <td class="py-1 border-b-1 border-r-1">
+                                {{ `${item.householdsAffected || "-"}` }}
+                            </td>
+                            <td class="py-1 border-b-1 border-r-1">
+                                {{ `${item.peopleAffected || "-"}` }}
+                            </td>
+                            <td class="py-1 border-b-1">
+                                {{ `${item.message || "-"}` }}
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+                <div v-else>
+                    Aucune information disponible sur les orientations des
+                    ménages
+                </div>
+            </div>
+        </template>
+    </DetailsPanel>
+</template>
+
+<script>
+import DetailsPanel from "#app/components/ui/details/DetailsPanel.vue";
+import DetailsPanelSection from "#app/components/ui/details/DetailsPanelSection.vue";
+
+export default {
+    components: { DetailsPanel, DetailsPanelSection },
+    props: {
+        town: {
+            type: Object
+        }
+    },
+    data() {
+        const {
+            closing_solutions: closingSolutions
+        } = this.$store.state.config.configuration;
+        return {
+            closingSolutions: closingSolutions.reduce((acc, item) => {
+                acc[item.id] = item.label;
+                return acc;
+            }, {})
+        };
+    },
+    computed: {
+        closedAt() {
+            return window.App.formatDate.apply(window, [this.town.closedAt]);
```

Comment:
Pas besoin d'utiliser la syntaxe `apply` ici : `window.app.formatDate(this.town.closedAt);`